src/Controllers/CategoryController.php

```diff
@@ -245,12 +245,15 @@ private function renderCategory($category, $params = [])
             /** @var OrderReturnController $orderReturnController */
             $orderReturnController = pluginApp(OrderReturnController::class);
 
+            RouteConfig::overrideCategoryId(RouteConfig::ORDER_RETURN, $category->id);
```

Comment:
Warum überschreibst du hier die CategoryId? Du fragst vorher in der if-Bedingung ab, ob die Id schon gesetzt ist? Das Überschreiben dürfte also nichts ändern.

Reply:
Ich habe mir das von anderen Stellen in der selben Klasse abgeschaut und habe das nach dem testen vergessen zu entfernen. Ich habe die Zeile wieder entfernt 👍